scrape_rec/settings.py

```diff
@@ -5,20 +5,18 @@
 
 ROBOTSTXT_OBEY = True
 
-DOWNLOAD_DELAY = 1
-
 ITEM_PIPELINES = {
     'scrape_rec.pipelines.NeighborhoodFinderPipeline': 100,
     'scrapy_jsonschema.JsonSchemaValidatePipeline': 300,
     'scrape_rec.pipelines.PostgresPipeline': 400
 }
 
 # Be nice!
-DOWNLOAD_DELAY = 3 # Average time between sending requests
-RANDOMIZE_DOWNLOAD_DELAY = True # Default for 0.5 to 1.5 times DOWNLOAD_DELAY
-AUTOTHROTTLE_ENABLED = True # Enable built-in autothrottle extension
-AUTOTHROTTLE_TARGET_CONCURRENCY = 1 # Number of requests to send in paralel
-CLOSESPIDER_ITEMCOUNT = 1000 # Stop after about 1000 items
+DOWNLOAD_DELAY = 5  # Average time between sending requests
+RANDOMIZE_DOWNLOAD_DELAY = True  # Default for 0.5 to 1.5 times DOWNLOAD_DELAY
+AUTOTHROTTLE_ENABLED = True  # Enable built-in autothrottle extension
+AUTOTHROTTLE_TARGET_CONCURRENCY = 1  # Number of requests to send in paralel
+CLOSESPIDER_ITEMCOUNT = 100  # Stop after about 1000 items
```

Comment:
DOWNLOAD_DELAY e cam mare, nu stiu daca ai beneficii daca astepti 5 in loc de 3 secunde
Si, 1000 acopera si o zi sau doua cand nu s-au strans date, as zice sa lasi pe 1000 cel putin cand rulezi prima data si daca nu sunt probleme e ok si 100 cred